kvmagent/kvmagent/plugins/tensorfusion/plugin.py

```
@@ -0,0 +1,354 @@
+                LibvirtEventManager.EVENT_SHUTDOWN,
+                LibvirtEventManager.EVENT_CRASHED,
+                LibvirtEventManager.EVENT_UNDEFINED,
+            ):
+                return
+
+            vm_uuid = dom.name()
+            count = self._service.destroy_workers_by_vm(vm_uuid)
+            if count > 0:
+                logger.info('TensorFusion: cleaned up %d workers for VM %s after lifecycle event %s' % (
+                    count, vm_uuid, event_str))
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)